calendar/Module.md

```diff
@@ -0,0 +1,94 @@
+## Overview
+Ballerina connector for Microsoft Outlook Calendar is connecting the Calendar API in Microsoft Graph via Ballerina language easily. 
+It provides capability to perform perform CRUD (Create, Read, Update, and Delete) operations on 
+[Event](https://docs.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0) &
+[Calendar](https://docs.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0) 
+stored in Microsoft OneDrive. 
+ 
+This module supports Microsoft Outlook Calendar API v1.0 version.
+ 
+### Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Create [Microsoft Outlook Account](https://outlook.live.com/owa/)
+* Obtaining tokens
+    Follow [this link](https://docs.microsoft.com/en-us/graph/auth-v2-user#authentication-and-authorization-steps) and obtain the client ID, client secret and refresh token.
```

Comment:
client ID, client secret, and refresh token.

Reply:
Shall we remove the trailing part? `and obtain the client ID, client secret and refresh token.`

Reply:
Corrected /Thanks